### PR TITLE
Add docker resource recommendation

### PR DESCRIPTION
**Pull request by muhwagwa**
Add docker resource recommendation
Rearrange beta docker docs to match the format of other ones

### CODE REVIEW

File: beta/docker/README.md [hunk 1]
```diff
@@ -357,6 +349,84 @@ For example, if you regenerate credentials for the automated user provisioning i
 
 A similar process can be used to update the values for any other Docker secrets used in your configuration.
 
+## Appendix: Resource recommendations
+
+The 1Password SCIM Bridge Pod should be vertically scaled when provisioning a large number of users or groups. Our default resource specifications and recommended configurations for provisioning at scale are listed in the below table:
```
Comment (scottisloud) [inline beta/docker/README.md:354]:
```suggestion
The 1Password SCIM Bridge container should be vertically scaled when provisioning a large number of users or groups. Our default resource specifications and recommended configurations for provisioning at scale are listed in the below table:
```

"Pod" would be Kubernetes-specific lingo.

**Review by scottisloud — APPROVED** [review]
Minor change suggested but otherwise LTGM!!

File: beta/docker/README.md (hunk 1 above)
Comment (muhwagwa) [reply beta/docker/README.md]: thanks for spotting that!

**Review by plttn — APPROVED** [review]
LGTM.

**Pull request merged**